api/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
 
 # Create your models here.
+
+class Task (models.Model):
+
+	STATUS_CHOICES = (
+		('UNASSIGNED', 'Unassigned'),
+		('WORKINGON', 'Working On'),
+		('COMPLETED', 'Completed'),
+		)
```

Comment:
we might want to add at least 1 more option, though: assigned (as in it's not being worked on yet) :wink: 
(yes, "at least" :grimacing: )